django/db/models/functions/datetime.py

```diff
@@ -80,6 +80,14 @@ class ExtractYear(Extract):
     lookup_name = 'year'
 
 
+class ExtractIsoYear(Extract):
+    """
+    Return the ISO week-numbering year based on ISO-8601, i.e., Jan 1st, 2017
```

Comment:
It might be better to describe how ISO-8601 or give a reference. I'm not sure that the example helps much. I think a description for `ExtractIsoYear` should be added to the docs as well.